src/estimagic/compat.py

```diff
@@ -0,0 +1,33 @@
+"""Compatibility module.
+
+Contains wrapper functions to handle compatibility issues between different versions of
+external libraries.
+
+"""
+
+from estimagic.config import IS_PANDAS_VERSION_NEWER_OR_EQUAL_TO_2_1_0
+
+
+def pandas_df_map(df, func, na_action=None, **kwargs):
```

Comment:
```suggestion
def pd_df_map(df, func, na_action=None, **kwargs):
```
Since `import pandas as pd` is the convention used throughout, that seems easier to parse.